library/swift/src/StatsClientImpl.swift

```diff
@@ -0,0 +1,19 @@
+@_implementationOnly import EnvoyEngine
+import Foundation
+
+/// Envoy implementation of StatsClient.
+@objcMembers
```

Comment:
This shouldn't be necessary since internal types don't need to be exposed to public objc consumers